source/introduction/what-is-ethereum.rst

```diff
@@ -1,42 +1,74 @@
-
-In Bitcoin's case the distributed database is conceived of as a table of account balances, a ledger, and transactions are transfers of the bitcoin token to facilitate trustless finance between individuals. But as bitcoin began attracting greater attention from developers and technologists, novel projects began to use the bitcoin network for purposes other than transfers of value tokens. Many of these took the form of "alt coins" - separate blockchains with cryptocurrencies of their own which improved on the original bitcoin protocol to add new features or capabilities. In late 2013, Ethereum's inventor Vitalik Buterin proposed that a single blockchain with the capability to be reprogrammed to perform any arbitrarily complex computation could subsume these many other projects.
-
-In 2014, Ethereum founders Vitalik Buterin, Gavin Wood and Jeffrey Wilcke began work on a next-generation blockchain that had the ambitions to implement a general, fully trustless smart contract platform.
+In 2014, Ethereum founders Vitalik Buterin, Gavin Wood and Jeffrey Wilcke began
+implementing Ethereum as a scalable, trustless smart contract platform.
 
 ================================================================================
 Ethereum Virtual Machine
 ================================================================================
 
-Ethereum is a programmable blockchain. Rather than give users a set of pre-defined operations (e.g. bitcoin transactions), Ethereum allows users to create their own operations of any complexity they wish. In this way, it serves as a platform for many different types of decentralized blockchain applications, including but not limited to cryptocurrencies.
+Unlike previous generations of blockchain technology which pre-defined operationd,
+Ethereum allows you to create on chain program operations of any complexity.
+In this way, it serves as a platform for many different types of decentralized
+blockchain applications, including cryptocurrencies.
+
+The Ethereum network protocols define the platform at the heart of which is the
+:ref:`Ethereum Virtual Machine ("EVM") <the-EVM>`, which can execute code of
+arbitrary algorithmic complexity.
+
+In computer science terms, Ethereum is "Turing complete", meaning that developers
```

Comment:
Hi Zelig...when i was changing this i considered using a formal definition of Turing Complete but found it esoteric and unhelpful in understanding what made Ethereum special.  Here, I'm just using the operational explanation of it as the previous writer had done.